x/auth/module.go

```diff
@@ -210,8 +210,7 @@ func ProvideAddressCodec(config *modulev1.Module) address.Codec {
 	return codecaddress.NewBech32Codec(config.Bech32Prefix)
 }
 
-//nolint:revive
-type AuthInputs struct {
+type Inputs struct {
```

Comment:
For consistency, we should keep the previous naming.
All modules follow this pattern. This never gets imported, so the linter complaining about a duplicate name if we import it as `auth.AuthInputs` is irrelevant.

Reply:
can do that